ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -460,6 +462,7 @@
 {%                 if port_channel_interface is arista.avd.defined %}
 {%                     set channel_group = ethernet_interface.channel_group.id | arista.avd.default("-") %}
 {%                     set isis_instance = port_channel_interface.isis_enable | arista.avd.default("-") %}
+{%                     set isis_bfd = port_channel_interface.isis_bfd | arista.avd.default("False") %}
```

Comment:
I think I would prefer a '-' as default value. Then it is clear if someone gave `false` as the input value vs. unset. (Same for all tables)

Reply:
Changed it to "-"